test/integration/authentication_test.rb

```diff
@@ -40,9 +40,15 @@ def silence_omniauth
     assert_equal(usercount + 1, User.count)
   end
 
-  test 'redirect to root path after login' do
+  test 'redirect to root path after login for normal users' do
```

Comment:
We've scrapped the test that anon users see the splash page - did we want to scrap that?

Reply:
I feel pretty medium about explicitly testing a page that has no access restrictions to make sure it has no access restrictions.

Since I'll be making changes to this PR, I'll go ahead and add one though as I don't feel strongly against it either. :)

Reply:
Added in static controller test